test/Fail/MissingDefinitions.err

```diff
@@ -1,11 +1,5 @@
-MissingDefinitions.agda:27,3-30,50
-The following names are declared but not accompanied by a
-definition: T₂, U₂, V₂, W₂
-MissingDefinitions.agda:5,1-18,4
-The following names are declared but not accompanied by a
-definition: Q, U, V, W, X
-MissingDefinitions.agda:22,3-23,4
-The following names are declared but not accompanied by a
-definition: A, B
-when scope checking the declaration
-  module AB where
```

Comment:
So we lost a couple of warnings here, I noticed now.
Not the world, but in general, we aim for maximizing the recoverable errors so that the interaction is more functional (hard errors abort the interaction).  I guess in this case, there was maybe not a good solution with feasible implementation effort.  This can be revisited should we decide to allow `mutual` in implicit mutual blocks.